docs/examples/Power Flow Example.ipynb

```diff
@@ -1404,16 +1403,16 @@
      "output_type": "stream",
      "text": [
       "Node data with invalid results\n",
-      "[[0.99940117 0.99268579 0.99452137]\n",
-      " [0.99934769 0.98622639 0.98935286]\n",
-      " [0.99928838 0.97965401 0.98409554]\n",
-      " [0.         0.         0.        ]\n",
-      " [0.99915138 0.96614948 0.97329879]\n",
-      " [0.99907317 0.95920586 0.96775071]\n",
-      " [0.9989881  0.95212621 0.96209647]\n",
-      " [0.         0.         0.        ]\n",
-      " [0.99879613 0.93753005 0.95044796]\n",
-      " [0.9986885  0.92999747 0.94444167]]\n",
+      "[[9.99401170e-001 9.92685785e-001 9.94521366e-001]\n",
+      " [9.99347687e-001 9.86226389e-001 9.89352855e-001]\n",
+      " [9.99288384e-001 9.79654011e-001 9.84095542e-001]\n",
+      " [6.95296562e-310 8.59448882e-312 6.95296562e-310]\n",
+      " [9.99151380e-001 9.66149483e-001 9.73298790e-001]\n",
+      " [9.99073166e-001 9.59205860e-001 9.67750710e-001]\n",
+      " [9.98988099e-001 9.52126208e-001 9.62096474e-001]\n",
+      " [8.59448882e-312 8.59448882e-312 6.95296562e-310]\n",
+      " [9.98796126e-001 9.37530046e-001 9.50447962e-001]\n",
+      " [9.98688504e-001 9.29997471e-001 9.44441670e-001]]\n",
```

Comment:
The results are the same, but I am very curious why did the formatting change so much.

Reply:
Because the garbage values were 0.0 before now they are some e-312

Reply:
but also the `e-001` ones. Just something funny I wanted to point out. I'll resolve.

Reply:
could be python3.12 vs python3.13 or windows vs unix or arm vs x86